src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,7 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+  let employeeEntry = employees.filter(employee => employee.name === employeeName);
+  return employeeEntry[0].role;
```

Comment:
Its convention to leave a clear line before the return statement to make it a bit easier to read:
```
  let employeeEntry = employees.filter(employee => employee.name === employeeName);

  return employeeEntry[0].role;
```